app/grandchallenge/algorithms/views.py

```diff
@@ -386,43 +389,54 @@ def get_context_data(self, *args, **kwargs):
         return context
 
 
-class AlgorithmResultsList(PermissionListMixin, ListView):
+class AlgorithmResultsList(PermissionListMixin, PaginatedTableListView):
     model = Result
     permission_required = (
         f"{Result._meta.app_label}.view_{Result._meta.model_name}"
     )
+    row_template = "algorithms/data_tables/result_list.html"
+    search_fields = [
+        "job__creator__username",
+        "job__image__name",
+        "images__files__file",
+        "comment",
+    ]
+    columns = [
+        "job__created",
+        "job__creator__username",
+        "job__image__name",
+        "visibility",
+        "images__files__file",
+        "comment",
+    ]
+    order_by = "job__created"
```

Comment:
Will this add a sort on unixtime?

Reply:
That's a good question, I'll look into it.

Reply:
It just sorts on the datetime field in postgres, but that effectively achieves the same.

Reply:
Nice, yes, this is much nicer!